### PR TITLE
Skip known errors or unsupported api in real-world api spec

### PR DESCRIPTION
**Pull request by shenie**
I've skipped 3.1.0 yamls which is going to be supported only after https://github.com/APIDevTools/swagger-parser/pull/171

Then I added what I can see are errors with the api to known errors list.

fwiw the spec is also failing on v10.0.2 presumably because the swagger yaml's has changed since last tag (assuming that the build was green when last tag was made) but I'm just trying to get main green asap so that it can get another tag release... so that I can get the z-schema cve fix....

### CODE REVIEW

**Comment by shenie** [comment]
@philsturgeon pls review

**Comment by philsturgeon** [comment]
If somebody can get the build to pas I’ll be happy to merge it. No time to dig into why it’s failing, still on a month like bike tour of Europe. Bad WiFi I’m the desert. I believe in you!

**Comment by shenie** [comment]
I've fixed the lint error. How do I trigger the build again? Because I'm seeing the following message at the bottom of my PR. 

'''
1 workflow awaiting approval
First-time contributors need a maintainer to approve running workflows. Learn more.
'''

**Comment by shenie** [comment]
I excluded another 3.1.0 yaml so the node 12 job should pass now.
The other failure is the browser job... but I can't tell what failed....

**Comment by shenie** [comment]
And it looks like github actions that runs the build still require approval every time I push a fix...

**Comment by philsturgeon** [comment]
@shenie sorry for this annoying github security feature, we've changed how tests run a little while ago and this is part of that. I'll be around my computer a bunch to click "Approve and Run" as much as needed. So far only two failing on ubuntu now!

**Comment by shenie** [comment]
Thanks.. working on it this morning.

**Comment by shenie** [comment]
Ok, pushed. `npm run test:node` passes on my laptop.

**Review by philsturgeon — APPROVED** [review]

**Pull request merged**
